File: js/1011.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
let a = parseFloat(lines.shift())


console.log("VOLUME =", ((4.0/3.0) * 3.14159 * Math.pow(a, 3)).toFixed(3))

File: c/PROBLEMS/1011.c
#include <stdio.h>
#include <math.h>

int main(){
    
    double a, pi=3.14159;

    scanf ("%lf", &a);
    printf("VOLUME = %.3lf\n", ((4.0/3.0) * pi * pow(a, 3)));

    return 0;
}